lib/extract-transforms/babel-globalize2-default-into-globalize.js

```diff
@@ -0,0 +1,26 @@
+var esprima = require( "esprima" );
```

Comment:
this is a duplicate file now (this was already in `extract-default-message-transforms` but not exported)